File: app/src/main/java/com/saienko/androidthings/barman/ui/InfoActivity.java
package com.saienko.androidthings.barman.ui;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import com.saienko.androidthings.barman.R;

public class InfoActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_info);
        findViewById(R.id.btnClose).setOnClickListener(view -> finish());
    }

    public static void start(Context context) {
        Intent intent = new Intent(context, InfoActivity.class);
        context.startActivity(intent);
    }
}

File: app/src/main/java/com/saienko/androidthings/barman/ui/InfoActivity.kt
package com.saienko.androidthings.barman.ui

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.saienko.androidthings.barman.R

class InfoActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_info)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)
    }

    override fun onSupportNavigateUp(): Boolean {
        finish()
        return true
    }

    companion object {

        fun start(context: Context) {
            val intent = Intent(context, InfoActivity::class.java)
            context.startActivity(intent)
        }
    }
}